app/views/cloud_network/edit.html.haml

```diff
@@ -1,4 +1,9 @@
-%form#form_div{:name => "angularForm", 'ng-controller' => "cloudNetworkFormController"}
+%form#form_div{:name => "angularForm",
+     'ng-controller' => "cloudNetworkFormController as vm",
```

Comment:
Remember to align these lines please, before unWIPping :).

(:name and 'ng-controller' (and the others) should start on the same column, and ideally so should the arrows.)